.github/workflows/build_wheels.yml

```
@@ -205,7 +205,7 @@ jobs:
           platforms: all
 
       - name: Build wheels
-        uses: pypa/cibuildwheel@v2.17.0
+        uses: pypa/cibuildwheel@v2.21.3
```

We could use
```suggestion
        uses: pypa/cibuildwheel@v2
```
so that we pick up minor updates automatically. What do you think?

---

make sense